composer.json

```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "magento/framework": "~102.0",
+        "magento/framework": "~102.0||~103.0",
         "magento/module-tax": "~100.3",
         "php": "~7.2.0||~7.3.0||~7.4.0",
         "symfony/console": "~4.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.5",
+        "phpunit/phpunit": "~6.5||~9.1",
```

Comment:
Hmm our tests will most probably break with version 9, right? Then we should not do this. Maybe it is better to switch to version 9 completely and update our tests as well?

Reply:
Opened an issue for this https://github.com/customgento/module-cli-tax-editor-m2/issues/4